### DISCUSSION TITLE
Deplying model

### DISCUSSION THREAD

**Discussion by somnathrakshit** [somnathrakshit/geograpy3#31] [Ideas]
Hello @WolfgangFahl, you might find this interesting. I have not used it before but I have seen lots of modern sites using it. This supports asynchronous call as well: [details](https://fastapi.tiangolo.com/async/), [video tutorial](https://youtu.be/Mw9etoRz0Ic).

**Comment by WolfgangFahl** [comment]
Thx - this is a very good hint
